### PR TITLE
ZEN-28949 Files can't be uploaded to dmd via HTTP PUT

### PR DESCRIPTION
**Pull request by ssOlexBaiko**
[Jira](https://jira.zenoss.com/browse/ZEN-28949)

### CODE REVIEW

**Comment by webhookbotuser** [comment]
Build finished.

File: Products/ZenUtils/patches/ofsmonkey.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from Products.ZenUtils.Utils import monkeypatch
+from zExceptions import MethodNotAllowed
+
+
+@monkeypatch(DTMLMethod)
+def PUT(self, REQUEST, RESPONSE):
+    """
+    Disable HTTP PUT for preventing upload to dmd without authentication
+    """
+    raise MethodNotAllowed, 'Method not supported for this resource.'
```
Comment (jpeacock-zenoss) [inline Products/ZenUtils/patches/ofsmonkey.py]: Please use the modern (and recommended) syntax:
```
raise MethodNotAllowed('Method not supported for this resource.')
```

Comment (ssOlexBaiko) [reply Products/ZenUtils/patches/ofsmonkey.py]: ok, sure

**Comment by webhookbotuser** [comment]
Build finished.

**Comment by webhookbotuser** [comment]
Build finished.

**Pull request merged**